src/components/Tabs/index.examples.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {storiesOf} from '@kadira/storybook'
+import Tabs from '.'
+import Card from '../Card'
+
+storiesOf('Tabs')
+  .addWithInfo(
+    'Info',
+    'Used to display sections of components',
+    () => (
+      <Tabs groups={[
+        {
+          title: 'Tab 1',
+          component: (
+            <Card>
+              <h1>Tab Content 1</h1>
+            </Card>
+          )
+        },
+        {
+          title: 'Tab 2',
+          component: (
+            <Card>
+              <h1>Tab Content 2</h1>
+            </Card>
+          )
+        }
+      ]}
+      />
+    )
+  )
```

Comment:
So, I updated all examples to use fixtures so that there isn't any manual content in examples. If you could update this in the same way that would be great. Example: https://github.com/eggheadio/egghead-ui/blob/master/src/components/List/index.examples.js.